resources/views/projects.blade.php

```diff
@@ -6,11 +6,80 @@
 @section('content')
 <div class="padded content">
 	<h1>Projects</h1>
+	<!-- Filters -->
+	<form class="ui form">
+		<!-- Search by name -->
+		<div class="field">
+			<label for="searchName">Name</label>
+			<input id="searchName" type="text" name="searchName" placeholder="e.g. A cool project">
+		</div>
+		<!-- Search by tag -->
+		<div class="field">
+			<label for="tags">Tags</label>
+			<select id="searchTags" name="tags" class="ui fluid search dropdown searchTags" multiple>
+				@foreach ($tags as $tag)
+					<option value={{ $tag->id }}>{{ $tag->name }}</option>
+				@endforeach
+			</select>
+		</div>
+		<button id="searchButton" type="button" class="ui primary submit button" onclick="filterProjects()">Search</button>
+	</form>
+	<!-- Project Cards -->
 	<div class="ui four stackable cards">
-		@for ($i = 0; $i < 10; $i++)
-			@projectCard(['projectImage' => 'https://source.unsplash.com/400x300/?project', 'projectName' => 'A cool project', 'category' => 'Videogames', 'projectShortDescription' => 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.', 'skillset' => ['Skill 0', 'Skill 1', 'Skill 2'], 'publicMilestone' => 'shipping'])
+		@foreach ($projects as $project)
+			@projectCard(['id'=> $project->id,'projectImage' => $project->photo, 'projectName' => $project->name, 'projectShortDescription' => $project->short_description, 'skillset' => $project->skills, 'labels' => $project->labels, 'publicMilestone' => 'shipping'])
 			@endprojectCard
-		@endfor
+		@endforeach
 	</div>
 </div>
 @endsection
+@section('scripts')
+<script>
+	$('.ui.dropdown').dropdown();
+	// npm install --save lodash.debounce OR yarn add lodash.debounce:
+	import _debounce from 'lodash.debounce';
+	// Event handler (do something on type):
+	$('#searchName').on('keyup', _debounce(filterProjects2, 200));
+	$('#searchTags').on('change', filterProjects2);
+	// Flatten out the object:
+	projects.forEach((project, index) => {
+		project.name = project.name.toLowerCase();
+		project.domid = index;
+		project.tags = project.tags.map(tag => {
+			return tag.name;
+		});
+	});
+	// DOM Elements (<projectCard>):
+	const $projectCards = $('.ProjectCard-container'); // Array
+	// projects[0] => $projectCards[0]
+	// Search by name, labels, or skills
+	// Tags => labels, skills
+	function filterProjects2() {
```

Comment:
la función se debe de llamar `filterProjects` en lugar de `filterProjects2` porque ya no exista la anterior

Reply:
Will be in next PR